### PR TITLE
fix(tap-agent): allow sender on allocation close

### PR DESCRIPTION
**Pull request by gusinacio**

### CODE REVIEW

File: tap-agent/src/agent/sender_account.rs [hunk 1]
```diff
@@ -45,6 +45,8 @@ pub enum SenderAccountMessage {
     GetSenderFeeTracker(ractor::RpcReplyPort<SenderFeeTracker>),
     #[cfg(test)]
     GetDeny(ractor::RpcReplyPort<bool>),
+    #[cfg(test)]
+    HasSchedulerEnabled(ractor::RpcReplyPort<bool>),
```
Comment (aasseman) [inline tap-agent/src/agent/sender_account.rs:49]:
```suggestion
    IsSchedulerEnabled(ractor::RpcReplyPort<bool>),
```
or
```suggestion
    HasScheduler(ractor::RpcReplyPort<bool>),
```

File: tap-agent/src/agent/sender_account.rs [hunk 2]
```diff
@@ -1538,4 +1573,69 @@ pub mod tests {
...
+        allocation.stop_and_wait(None, None).await.unwrap();
+        allocation_handle.await.unwrap();
+
+        tokio::time::sleep(Duration::from_millis(100)).await;
+
+        // should remove the block and the retry
+        let deny = call!(sender_account, SenderAccountMessage::GetDeny).unwrap();
+        assert!(!deny, "should be unblocked");
+
+        let scheuduler_enabled =
```
Comment (aasseman) [inline tap-agent/src/agent/sender_account.rs:1634]:
```suggestion
        let scheduler_enabled =
```

**Review by aasseman — CHANGES_REQUESTED** [review]
LGTM, just a couple of typos

File: tap-agent/src/agent/sender_account.rs (hunk 2 above)
Comment (gusinacio) [reply tap-agent/src/agent/sender_account.rs:1634]: Done

File: tap-agent/src/agent/sender_account.rs (hunk 1 above)
Comment (gusinacio) [reply tap-agent/src/agent/sender_account.rs]: Done

**Review by aasseman — APPROVED** [review]

**Pull request merged**